### ISSUE TITLE
H265 video to grapFrame setFrameNum(frameNum) to The specified Frame have a error:Could not find ref with POC 0

### ISSUE THREAD

**Issue by duck123456** [bytedeco/javacv#1227]
ff.setFrameNumber(num);
     Frame frame = ff.grabKeyFrame();

log:
[hevc @ 000000001e36a880] Could not find ref with POC 0

Thanks

**Comment by duck123456** [comment]
avformat_seek_file(oc, -1, Long.MIN_VALUE, timestamp, Long.MAX_VALUE, AVSEEK_FLAG_BACKWARD) this  problem，But don't know how to solve it

**Comment by duck123456** [comment]
while (this.timestamp > timestamp + 1 && grabFrame(true, true, false, false) != null && count++ < 1000) {
                    // flush frames if seeking backwards
                } 

haha Problem solved !  keyframes set  true  grabFrame(true, true, false, true)

**Comment by saudet** [comment]
That's probably just a warning, just ignore it.

**Comment by duck123456** [comment]
> That's probably just a warning, just ignore it.
It blurs out all the images taken out are blurred, except for the first one. Is this related to the key frame?

**Comment by saudet** [comment]
Probably, we can try to use your fix for calls to grabFrame(). Could you send a pull request?

**Comment by saudet** [comment]
Actually, it shouldn't matter how we call grabFrame(). It just sounds like your video files is corrupted. Try to reencode it.

**Comment by duck123456** [comment]
[https://github.com/bytedeco/javacv/blob/master/src/main/java/org/bytedeco/javacv/FFmpegFrameGrabber.java#L717](url)
What does this step do? Here's the problem

**Comment by duck123456** [comment]
> Probably, we can try to use your fix for calls to grabFrame(). Could you send a pull request?

https://github.com/bytedeco/javacv/blob/master/src/main/java/org/bytedeco/javacv/FFmpegFrameGrabber.java#L717

this keyframes is false ，The question is whether it is relevant？

**Comment by saudet** [comment]
No, I don't think so. I think you just have a bad file. Try another one.

**Comment by duck123456** [comment]
The file is normal，Try othor some video file  same problem.   use h264 encode video file  is normal,but use h165 encode video file error occurred

**Comment by duck123456** [comment]
> No, I don't think so. I think you just have a bad file. Try another one.

The file is normal，Try othor some video file same problem. use h264 encode video file is normal,but use h165 encode video file error occurred

**Comment by duck123456** [comment]
if not setFrameNumber(num) is normal,but i think to seek getFrame

**Comment by duck123456** [comment]
> No, I don't think so. I think you just have a bad file. Try another one.

The file is normal，Try othor some video file same problem. use h264 encode video file is normal,but use h265 encode video file error occurred

**Comment by saudet** [comment]
Ok, so this sounds like an issue with the decoder maybe. Could you report this issue upstream?

**Comment by saudet** [comment]
This appears to be an old bug in FFmpeg that has already been fixed though:
https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/10bc4c3a7df7bb26303067b97311b7eeedfd453e
Please upgrade to JavaCV 1.5!

**Comment by duck123456** [comment]
> This appears to be an old bug in FFmpeg that has already been fixed though:
> https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/10bc4c3a7df7bb26303067b97311b7eeedfd453e
> Please upgrade to JavaCV 1.5!

Thanks I have updated javaCV1.5！Previous versions were 1.5

**Comment by saudet** [comment]
Ok, good, so it works well now?

**Comment by duck123456** [comment]
> Ok, good, so it works well now?

It still has the same problem

**Comment by saudet** [comment]
Ok, so ask the authors of that patch. They might know why this is happening.

**Comment by duck123456** [comment]
> Ok, so ask the authors of that patch. They might know why this is happening.

Ok, thank you!
